WooCommerce/Classes/App Intents/CreateOrderAppIntent.swift

```diff
@@ -0,0 +1,28 @@
+import AppIntents
+import Yosemite
+import Foundation
```

Comment:
Same as with `CollectPaymentAppIntent`, it doesn't seem we need to import these two modules:
```suggestion
```